### PR TITLE
Don't expect argless call to CanvasRenderingContext2D.setTransform() to throw

### PR DESCRIPTION
**Pull request by wisniewskit**
Addresses https://github.com/w3c/web-platform-tests/issues/7357

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PENDING

Started: None
Finished: None





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/18568)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/275723061)

File: 2dcontext/conformance-requirements/2d.missingargs.html [hunk 1]
```diff
@@ -33,7 +33,6 @@
     assert_throws(new TypeError(), function() { ctx.transform(1, 0, 0, 1, 0); });
 }
 if (ctx.setTransform) {
-    assert_throws(new TypeError(), function() { ctx.setTransform(); });
```
Comment (bzbarsky) [inline 2dcontext/conformance-requirements/2d.missingargs.html]: Can we instead test that it behaves like passing `{}`?

**Comment by zcorpan** [comment]
The top of the file says 
```
<!-- DO NOT EDIT! This test has been generated by tools/gentest.py. -->
```
So this change will be overwritten when someone else changes the source files and rebuilds.

Edit /2dcontext/tools/tests2d.yaml and run ./build.sh in that directory.

**Comment by zcorpan** [comment]
-> #7447

**Pull request closed without merging**